### ISSUE TITLE
Make qrbox resizable.

### ISSUE THREAD

**Issue by ValkoBob** [mebjas/html5-qrcode#585]
Make QRScanner resizable like a swiper, that makes sight of scanner to increase their width and height.
i.e., if qr-code, what we want to scan, is away from our device, and we would like to resize sight of scanner, that help us to focus on this qr-code.

**Comment by mebjas** [comment]
Would having a zoom slider help instead?

**Comment by ValkoBob** [comment]
> Would having a zoom slider help instead?

No, It needs to be, if it`s possible, a resizable sight in scanner.
